.happy/terraform/modules/sfn/main.tf

```diff
@@ -58,7 +58,11 @@ resource "aws_sfn_state_machine" "state_machine" {
               },
               {
                 "Name": "DATASET_ID",
-                 "Value.$": "$.dataset_id"
+                "Value.$": "$.dataset_id"
+              },
+              {
+                "Name": "COLLECTION_ID",
```

Comment:
not easily, since this is actually the collection version ID. thankfully, we actually already pass in the collection version ID as 'collection ID' when we trigger an upload job. we just haven't been using it until now

Reply:
see https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/backend/layers/thirdparty/step_function_provider.py#L36

Reply:
correction: we've been using it for logging in the handle errors step. This is just leveraging it elsewhere since we already have access to it